homeassistant/components/mqtt/__init__.py

```diff
@@ -91,6 +91,7 @@
     MqttCommandTemplate,
     MqttData,
     MqttValueTemplate,
+    MqttValueTemplateException,
```

Comment:
Why do we export this, do we expect other integrations to import it?

Reply:
Remove the import.